core/domain/classifier_services.py

```diff
@@ -472,6 +515,13 @@ def load_examples(self, examples):
 
     def predict_label_for_doc(self, d):
         """Returns the predicted label from a doc's prediction report.
+
+        Args:
+            d: int. A doc id from a prediction report (see example in
```

Comment:
**predict_label_for_doc** only takes a doc id. The caller will get this id from the prediction report, since the docs are represented as lists of word ids internally and there's no direct method for getting a doc id. I'm not sure myself about the docstring.